diff -urN inet6-apps-0.36.orig/ftpd/ftpd.8 inet6-apps-0.36/ftpd/ftpd.8
--- inet6-apps-0.36.orig/ftpd/ftpd.8	Sun May 23 12:32:52 1999
+++ inet6-apps-0.36/ftpd/ftpd.8	Sun May 23 12:33:02 1999
@@ -85,13 +85,13 @@
 .El
 .Pp
 The file
-.Pa /etc/nologin
+.Pa /etc/ftp/nologin
 can be used to disable ftp access.
 If the file exists,
 .Nm
 displays it and exits.
 If the file
-.Pa /etc/ftpwelcome
+.Pa /etc/ftp/welcome
 exists,
 .Nm
 prints it before issuing the 
@@ -267,14 +267,16 @@
 account in this directory.
 .El
 .Sh FILES
-.Bl -tag -width /etc/ftpwelcome -compact
-.It Pa /etc/ftpusers
+.Bl -tag -width /etc/ftp/welcome -compact
+.It Pa /etc/ftp/users
 List of unwelcome/restricted users.
-.It Pa /etc/ftpwelcome
+.It Pa /etc/ftp/welcome
 Welcome notice.
-.It Pa /etc/motd
+.It Pa /etc/ftp/motd
 Welcome notice after login.
-.It Pa /etc/nologin
+.It Pa /etc/ftp/motd-6
+Welcome notice after login through IPv6.
+.It Pa /etc/ftp/nologin
 Displayed and access refused.
 .El
 .Sh SEE ALSO
diff -urN inet6-apps-0.36.orig/ftpd/pathnames.h inet6-apps-0.36/ftpd/pathnames.h
--- inet6-apps-0.36.orig/ftpd/pathnames.h	Sun May 23 12:32:52 1999
+++ inet6-apps-0.36/ftpd/pathnames.h	Sun May 23 12:35:30 1999
@@ -35,10 +35,13 @@
 
 #include <paths.h>
 
-#define	_PATH_FTPUSERS		"/etc/ftpusers"
-#define	_PATH_FTPWELCOME	"/etc/ftpwelcome"
+#define	_PATH_FTPUSERS		"/etc/ftp/users"
+#define	_PATH_FTPWELCOME	"/etc/ftp/welcome"
+#ifndef _PATH_NOLOGIN
+#define	_PATH_NOLOGIN		"/etc/ftp/nologin"
+#endif
 #if INET6
-#define	_PATH_FTPLOGINMESG	"/etc/ftp.motd"
+#define	_PATH_FTPLOGINMESG	"/etc/ftp/motd-6"
 #else /* INET6 */
-#define	_PATH_FTPLOGINMESG	"/etc/motd"
+#define	_PATH_FTPLOGINMESG	"/etc/ftp/motd"
 #endif /* INET6 */
diff -urN inet6-apps-0.36.orig/ftpd/pathnames.h~ inet6-apps-0.36/ftpd/pathnames.h~
--- inet6-apps-0.36.orig/ftpd/pathnames.h~	Thu Jan  1 01:00:00 1970
+++ inet6-apps-0.36/ftpd/pathnames.h~	Sun May 23 12:35:15 1999
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 1989, 1993
+ *	The Regents of the University of California.  All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. All advertising materials mentioning features or use of this software
+ *    must display the following acknowledgement:
+ *	This product includes software developed by the University of
+ *	California, Berkeley and its contributors.
+ * 4. Neither the name of the University nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ``AS IS'' AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED.  IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+ * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ *
+ *	@(#)pathnames.h	8.1 (Berkeley) 6/4/93
+ */
+
+#include <paths.h>
+
+#define	_PATH_FTPUSERS		"/etc/ftp/users"
+#define	_PATH_FTPWELCOME	"/etc/ftp/welcome"
+#ifndef
+#define	_PATH_NOLOGIN		"/etc/ftp/nologin"
+#endif
+#if INET6
+#define	_PATH_FTPLOGINMESG	"/etc/ftp/motd-6"
+#else /* INET6 */
+#define	_PATH_FTPLOGINMESG	"/etc/ftp/motd"
+#endif /* INET6 */
