diff -Naur tftpd-1.2a1/Makefile.in ../tftpd-1.2a1/Makefile.in
--- tftpd-1.2a1/Makefile.in	Wed Jan 29 03:19:23 1997
+++ ../tftpd-1.2a1/Makefile.in	Wed Jul 29 15:46:03 1998
@@ -27,7 +27,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 # Pathname of directory to install the binary
-BINDEST = @sbindir@
+BINDEST = @libexecdir@
 # Pathname of directory to install the man page
 MANDEST = @mandir@
 
diff -Naur tftpd-1.2a1/tftpsubs.c ../tftpd-1.2a1/tftpsubs.c
--- tftpd-1.2a1/tftpsubs.c	Wed Jan 29 03:19:01 1997
+++ ../tftpd-1.2a1/tftpsubs.c	Wed Jul 29 15:51:33 1998
@@ -145,7 +145,7 @@
 	b = &rp->bfs[rp->nextone];		/* look at "next" buffer */
 	if (b->counter != BF_FREE) {
 		/* Restore possible errno */
-		errno = b->errno;
+		 /* errno = b->errno; */
 		return;
 	}
 	rp->nextone = !rp->nextone;		/* "incr" next buffer ptr */
@@ -155,7 +155,7 @@
 	if (!convert) {
 		errno = 0;
 		b->counter = read(fileno(f), dp->th_data, SEGSIZE);
-		b->errno = errno;
+	/*	b->errno = errno; */
 		return;
 	}
 
@@ -185,7 +185,7 @@
 		b->counter = -1;
 	else
 		b->counter = (int)(p - dp->th_data);
-	b->errno = errno;
+	/* b->errno = errno; */
 }
 
 #ifdef ALLOW_WRITE
diff -Naur tftpd-1.2a1/tftpsubs.h ../tftpd-1.2a1/tftpsubs.h
--- tftpd-1.2a1/tftpsubs.h	Tue Sep 13 09:29:51 1994
+++ ../tftpd-1.2a1/tftpsubs.h	Wed Jul 29 15:49:43 1998
@@ -37,7 +37,6 @@
 
 struct bf {
 	int counter;		/* size of data in buffer, or flag */
-	int errno;		/* errno after read */
 	char buf[PKTSIZE];	/* room for data packet */
 };
 
